docs/how_to/query_interface.rst

```diff
@@ -55,4 +55,5 @@ Resources
    /guides/primer/usage_pattern.md
    /how_to/index_structs/composability.rst
    /how_to/query/query_transformations.md
-   /how_to/query/node_postprocessor.md
\ No newline at end of file
+   /how_to/query/node_postprocessor.md
+   /how_to/query/RouterQueryEngine.ipynb
```

Comment:
Is there a way we can categorize this in a new section? 

e.g. "Tutorials", "Tutorial Notebooks", or "Guides"

Reply:
Removed from this PR. Landing with just the config changes.